### ISSUE TITLE
[Visual Studio Code - Project Manager] Unable to load remote project fold when there's "." in the folder path

### ISSUE THREAD

**Issue by sunrisewestern** [raycast/extensions#5411]
### Extension

https://www.raycast.com/MarkusLanger/vscode-project-manager

### Description

When connecting to remote project, if the folder path contains ".", it will only connect to the ssh server but will not open the project fold. This is a bug of vscode CLI but can be easily solved by just adding a "/" at the end of path.

**Comment by MarkusLanger** [comment]
@sunrisewestern Could you check if the code in the branch fixes your problem and if it also works for projects without a "." in the path
https://github.com/MarkusLanger/extensions/tree/5411

**Comment by sunrisewestern** [comment]
> @sunrisewestern Could you check if the code in the branch fixes your problem and if it also works for projects without a "." in the path https://github.com/MarkusLanger/extensions/tree/5411

Yes. it fixed my problem. And works on normal projects without dot.

**Comment by MarkusLanger** [comment]
PR created #5413

**Comment by sunrisewestern** [comment]
@raycastbot close this issue

**Comment by pernielsentikaer** [comment]
Strange, let me check why it didn't work

**Comment by patroza** [comment]
@MarkusLanger @pernielsentikaer sadly this breaks search in vs code remote over ssh,
also my projects show up double now in vs code recently used. See screenshot:

<img width="378" alt="Screenshot 2023-03-24 at 09 58 06" src="https://user-images.githubusercontent.com/42661/227472700-b48472cc-1b6d-475e-958b-830baa40af62.png">
